db/schema.rb

```diff
@@ -10,10 +10,33 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_07_18_203652) do
+ActiveRecord::Schema[7.0].define(version: 2023_07_24_160557) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "guest_rooms", id: false, force: :cascade do |t|
+    t.bigint "room_id", null: false
+    t.bigint "guest_id", null: false
+  end
+
+  create_table "guests", force: :cascade do |t|
+    t.string "name"
+    t.integer "nights"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "room_id"
+    t.index ["room_id"], name: "index_guests_on_room_id"
+  end
+
+  create_table "hotel_rooms", force: :cascade do |t|
+    t.bigint "hotel_id", null: false
+    t.bigint "room_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["hotel_id"], name: "index_hotel_rooms_on_hotel_id"
+    t.index ["room_id"], name: "index_hotel_rooms_on_room_id"
+  end
+
```

Comment:
You do not need this table. I see you got your model set up for it, but only `GuestRoom` was the necessary join table.